mayhem/c_fuzzer.mayhemfile

```diff
@@ -0,0 +1,5 @@
+project: tree-sitter
+target: c_fuzzer
+
+cmds:
```

Comment:
Specifying dictionaries in Mayhemfiles is done with the `dictionary` field:
```
cmds:
  - cmd: <some command>
  {...}
  dictionary: /path/to/my/dictionary
```

Specifying corpora is done with the `tests` field:

```
tests:
  - file:///absolute/path/to/my/corpus
```
See https://mayhem.forallsecure.com/docs/reference/mayhemfile/#dictionary and https://mayhem.forallsecure.com/docs/reference/mayhemfile/#tests for more info.